### ISSUE TITLE
doesn't load on october cms 2

### ISSUE THREAD

**Issue by shiftaltd** [FlusherDock1/EditorJS#19]
The editor doesn't load on october cms 2

the console returns many errors
`loader.js:27 Uncaught Error: Can only have one anonymous define call per script file`

A quick fix is to add

```
if("function" == typeof define && define.amd) {
    delete define.amd;
}
```
at the top of formwidgets/editorjs/assets/js/editor.js

https://stackoverflow.com/questions/55057425/can-only-have-one-anonymous-define-call-per-script-file

**Comment by FlusherDock1** [comment]
@shiftaltd thanks! I'll test it ASAP.

**Comment by FlusherDock1** [comment]
Added your fix in 1.7.5. Thanks!
